src/sidebar/components/test/new-note-btn-test.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+var angular = require('angular');
+
+var events = require('../../events');
+var util = require('../../directive/test/util');
+
+describe('newNoteBtn', function () {
+  var $rootScope;
+  var sandbox = sinon.sandbox.create();
+  var fakeAnnotationUI = {
+    frames: sinon.stub().returns([{ id: null, uri: 'www.example.org'}, { id: '1', uri: 'www.example.org'}]),
+  };
+
+  before(function () {
+    angular.module('app', [])
+      .component('selectionTabs', require('../selection-tabs'))
+      .component('newNoteBtn', require('../new-note-btn'));
+  });
+
+  beforeEach(function () {
+    var fakeFeatures = {
+      flagEnabled: sinon.stub().returns(true),
+    };
+    var fakeSettings = { theme: 'clean' };
+
+    angular.mock.module('app', {
+      annotationUI: fakeAnnotationUI,
+      features: fakeFeatures,
+      settings: fakeSettings,
+    });
+
+    angular.mock.inject(function (_$componentController_, _$rootScope_) {
+      $rootScope = _$rootScope_;
+    });
+  });
+
+  afterEach(function() {
+    sandbox.restore();
+  });
+
+  it('should display the new note button when the notes tab is active', function () {
+    var elem = util.createDirective(document, 'selectionTabs', {
```

Comment:
These are tests for the `selectionTabs` component, so they should be in the relevant test module ("selection-tabs-test.js"). I think the code will work there without any changes - the tests just need to be moved.